tools/cli/commands.js

```diff
@@ -1387,6 +1382,8 @@ main.registerCommand({
     return deploy.changeAuthorized(site, "add", options.add);
   } else if (options.remove) {
     return deploy.changeAuthorized(site, "remove", options.remove);
+  } else if (options.transfer) {
+    return deploy.changeAuthorized(site, "transfer", options.transfer);
```

Comment:
Oh, I thought we said that `transfer` would do the same as `add`? At least make sure @pcarrier knows that we'll have an explicit new `transfer` call that should be already functional when we ship Meteor 1.3.